src/scripts/oauth_tables_script.sql

```diff
@@ -0,0 +1,93 @@
+
+DROP TABLE IF EXISTS app_clients CASCADE;
+
+CREATE TABLE app_clients(
+             id BIGSERIAL NOT NULL,
+             client_name VARCHAR(50) NOT NULL,
+             client_id VARCHAR(500) NOT NULL,
+             client_secret VARCHAR(500) NOT NULL,
+             expiration_time TIMESTAMP WITHOUT TIME ZONE,
+             entity_status status_enum NOT NULL,
+             created_by BIGINT NOT NULL,
+             created_date TIMESTAMP WITHOUT TIME ZONE,
+             updated_by BIGINT NOT NULL,
+             updated_date TIMESTAMP WITHOUT TIME ZONE,
+             PRIMARY KEY (id)
+);
+
+DROP TYPE IF EXISTS scope_type_enum CASCADE;
+
+CREATE TYPE scope_type_enum AS ENUM ('READ', 'WRITE', 'READ/WRITE');
+
+DROP TABLE IF EXISTS client_users CASCADE;
+
+CREATE TABLE app_client_users(
+             id BIGSERIAL NOT NULL,
+             app_client_id BIGINT NOT NULL,
+             user_name VARCHAR(50) NOT NULL,
+             user_scope scope_type_enum NOT NULL,
```

Comment:
No need to do this here, this means that every combination of client id and user has one scope. Please create a table to relate clients and scopes for that client.

Reply:
done.